public/app/features/inspector/InspectJSONTab.tsx

```diff
@@ -73,6 +74,11 @@ export class InspectJSONTab extends PureComponent<Props, State> {
     };
   }
 
+  componentDidMount() {
+    // when opening the inspector we want to report the interaction
+    reportInteraction(`grafana_panel_inspect_json_panelJSON_clicked`);
```

Comment:
@axelavargas- just a nit regarding interactions that belong to the same domain. I think this is a good pattern to have those interaction consistent prefixes: https://github.com/grafana/grafana/blob/main/public/app/features/search/page/reporting.ts#L15-L16

I think we could have a higher order 
```function reportPanelInspectInteraction(name: string) => reportInteraction(`grafana_panel_inspect_json_${name}`, ... )```